recipe/meta.yaml

```diff
@@ -1,15 +1,15 @@
-{% set version = "0.24.0" %}
+{% set version = "0.22.0" %}
```

Comment:
If this is a 1-time thing I'm OK merging it on master and letting the bot "fix it" later. Do you need to maintain the 0.22.0 version @sodre? If so we need a branch.